src/libs/schedule.ts

```diff
@@ -151,106 +127,151 @@ const genericBlocks: Record<
 	}
 };
 
+function isOnLateStartList(date: Date) {
+	const lateStartList = lateStarts as DateList;
+	for (const lateStartDate of lateStartList.date) {
+		const lateStartMoment = moment(lateStartDate, 'MM/DD/YYYY');
+		if (moment(date).isSame(lateStartMoment)) {
+			return true;
+		}
+	}
+	return false;
+}
+
+function removeCertainLunchBlocks(
+	daySchedule: blockSchedule.BlockFormats,
+	keepAemsh: boolean,
+	keepHswl: boolean,
+	keepDefault: boolean
```

Comment:
Consider refactoring this into a parameters object `{ keepAemsh, keepHswl, keepDefault }`, as it's a bit confusing when called.